### PR TITLE
Adding our one-on-one guide

### PR DESCRIPTION
**Pull request by eiso**
Moving this from a Google Doc to be public information since it's nice for people to see how we like to structure our one-on-one meetings.

### CODE REVIEW

File: talent/one_on_one_guide.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+**source{d} One-On-One’s Guide**
+
+Below is the guide we use at source{d} for one-on-one meetings between you and your manager. They are guiding questions and its okay if someone choses to adapt them.
+
+**Schedule:**
+
+* One-on-one's are to be held every at a minimum every 3 months
```
Comment (Serabe) [inline talent/one_on_one_guide.md]: s/every at/at/

File: talent/one_on_one_guide.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
+**source{d} One-On-One’s Guide**
```
Comment (Serabe) [inline talent/one_on_one_guide.md]: Can we use markdown headers? # for level 1, ## for level 2 and so until level 6. This applies to the rest of the document.

File: talent/one_on_one_guide.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+Part III. Team improvement
+
+* What do you believe we can do better as a team? 
+
+Part IV. Feedback on performance
+
+* Praise first: what was well done and why you think so. It is important to give concrete examples here of deliverables/projects etc. you believe went well
+
+* Downsides: always mention the negative followed by actions to improve it. Before your meeting be sure you have prepared an action plan (set of bullet points on how you will together improve things) to tackle the downsides and FOLLOW-UP on it until the following one-on-one.
```
Comment (Serabe) [inline talent/one_on_one_guide.md]: Don't use capital letter to emphasize. Use bold or italic :)

**Comment by eiso** [comment]
Thank you @Serabe, made the changes.

**Comment by bzz** [comment]
Thank you for sharing it!

**Comment by eiso** [comment]
@bzz we've been using this format for years. The only major change is that one-on-one's started being moved to once every 3 months instead of monthly, primarily because of the # of people now under @mcuadros.

File: talent/one_on_one_guide.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
+# source{d} One-On-One’s Guide
+
+Below is the guide we use at source{d} for one-on-one meetings between you and your manager. They are guiding questions and its okay if someone choses to adapt them.
```
Comment (abeaumont) [inline talent/one_on_one_guide.md]: s/its/it's/

File: talent/one_on_one_guide.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+* What do you believe we can do better as a team? 
+
+### Part IV. Feedback on performance
+
+* Praise first: what was well done and why you think so. It is important to give concrete examples here of deliverables/projects etc. you believe went well
+
+* Downsides: always mention the negative followed by actions to improve it. Before your meeting be sure you have prepared an action plan (set of bullet points on how you will together improve things) to tackle the downsides and **follow up** on it until the following one-on-one. 
+
+### Part V. Career Development
```
Comment (erizocosmico) [inline talent/one_on_one_guide.md]: should this be every 3 months? It seems such a short time period to have changed your mind about this

Comment (eiso) [inline talent/one_on_one_guide.md]: It's more about following up. Someone says I want to be "X in 5 years". Then you discuss if during the last 3 months they did anything to help get there. I should clarify this.

**Comment by erizocosmico** [comment]
Ah, okay, cool.
On Thu, 6 Jul 2017 at 13:23, Eiso Kant <notifications@github.com<mailto:notifications@github.com>> wrote:

@eiso commented on this pull request.

________________________________

In talent/one_on_one_guide.md<https://github.com/src-d/guide/pull/56#discussion_r125873214>:

> +
+* Please let me know where you believe you can still improve? (it is important this becomes part of the action plan on how to improve)
+
+* If you could rate your happiness from 1-10? If not 10, why not and what can we do to improve it
+
+### Part III. Team improvement
+
+* What do you believe we can do better as a team?
+
+### Part IV. Feedback on performance
+
+* Praise first: what was well done and why you think so. It is important to give concrete examples here of deliverables/projects etc. you believe went well
+
+* Downsides: always mention the negative followed by actions to improve it. Before your meeting be sure you have prepared an action plan (set of bullet points on how you will together improve things) to tackle the downsides and **follow up** on it until the following one-on-one.
+
+### Part V. Career Development


It's more about following up. Someone says I want to be "X in 5 years". Then you discuss if during the last 3 months they did anything to help get there. I should clarify this.

**Pull request merged**